flintrock/flintrock.py

```diff
@@ -297,6 +305,21 @@ def launch(
         raise UnsupportedProviderError(provider)
 
 
+def get_last_commit_sha(git_repository: str):
+    """
+    Retrieve the last commit's SHA from a repository hosted on Github.
+    """
+    parsed_url = urllib.parse.urlparse(git_repository)
+    if parsed_url.netloc != "github.com":
+        raise UsageError('Error: Retrieving the last commit\'s SHA only works with '
```

Comment:
Nit: As noted before, I prefer this string be encapsulated by double quotes since it is prose, and the opposite goes for `"github.com"`.

Also, I'm not totally sure about the terminology, but I think "the latest commit" is sufficient. No need to mention SHA.